apps/app-frontend/src/components/ui/settings/AppearanceSettings.vue

```diff
@@ -43,51 +43,23 @@ watch(
       </p>
     </div>
 
-    <Toggle
-      id="advanced-rendering"
-      :model-value="themeStore.advancedRendering"
-      :checked="themeStore.advancedRendering"
-      @update:model-value="
-        (e) => {
-          themeStore.advancedRendering = e
-          settings.advanced_rendering = themeStore.advancedRendering
```

Comment:
this one no longer seems to set settings.advanced_rendering

I haven't checked but I assume this probably changes behavior?